contributing/development/core_and_modules/core_types.rst

```diff
@@ -103,36 +103,12 @@ which are equivalent to new, delete, new[] and delete[].
 memnew/memdelete also use a little C++ magic and notify Objects right
 after they are created, and right before they are deleted.
 
-For dynamic memory, the PoolVector<> template is provided. PoolVector is a
-standard vector class, and is very similar to vector in the C++ standard library.
-To create a PoolVector buffer, use this:
-
-.. code-block:: cpp
-
-    PoolVector<int> data;
-
-PoolVector can be accessed using the [] operator and a few helpers exist for this:
-
-.. code-block:: cpp
-
-    PoolVector<int>::Read r = data.read()
-    int someint = r[4]
-
-.. code-block:: cpp
-
-    PoolVector<int>::Write w = data.write()
-    w[4] = 22;
-
-These operations allow fast read/write from PoolVectors and keep it
-locked until they go out of scope. However, PoolVectors should be used
-for small, dynamic memory operations, as read() and write() are too slow for a
-large amount of accesses.
+For dynamic memory, use Vector<>.
```

Comment:
This page should also mention [`LocalVector`](https://github.com/godotengine/godot/blob/master/core/templates/local_vector.h), and [`Packed*Vector`](https://github.com/godotengine/godot/blob/0eadbdb5d0709e4e557e52377fa075d3e2f0ad1f/core/variant/variant.h#L73-L83).

- `Vector<>` - copy-on-write and can be safely passed via public API.
- `Packed*Vector` - aliases for specific `Vector<*>` types accessible via GDScript.
- `LocalVector<>` - non COW version, with less overhead, for the internal use.

Reply:
Yes, `For dynamic memory, use Vector<>` is correct, and in general a safe option to use, so it's OK as is.

Reply:
If it's OK as a bugfix, then let's merge this one as-is.

@esainane, you're welcome to document the other options mentioned here if you want to, as another PR, after this one is merged. If you don't feel knowledgeable enough, that is also OK.

Reply:
Sure, I should be able to do something proper this Sunday.

Reply:
I think I have a suitable follow up ready to go, in case people were waiting?

Reply:
It's fine to bundle it into this PR, since it's not actually merged yet.

(I thought we might have merged this already, but without an explicit second approval from an experienced engine dev I ended up not merging it yet)